portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileOrder.testcase

```diff
@@ -162,7 +162,7 @@ definition {
 
 		AssertElementPresent(locator1 = "DocumentsAndMedia#SORTED_DOWN_ICON");
 
-		ManagementBar.viewFilterAndOrder(menuItem = "Modified Date");
+		ManagementBar.viewFilterAndOrderItem(menuItem = "Modified Date");
```

Comment:
The macro has been changed because the aim of this test is not to check if the item is active but only to see if it is present in the dropdown.